src/features/gtl/api/gtl-playlist-api.native.test.ts

```diff
@@ -0,0 +1,391 @@
+                    updatedAt: '2023-09-14T14:29:07.319Z',
+                    environment: {
+                      sys: {
+                        id: 'testing',
+                        type: 'Link',
+                        linkType: 'Environment',
+                      },
+                    },
+                    revision: 2,
+                    contentType: {
+                      sys: {
+                        type: 'Link',
+                        linkType: 'ContentType',
+                        id: 'displayParameters',
+                      },
+                    },
+                    locale: 'en-US',
+                  },
+                  fields: {
+                    title: 'Jeunesse',
+                    layout: 'two-items',
+                    minOffers: 1,
+                  },
+                },
+                {
+                  metadata: {
+                    tags: [],
+                  },
+                  sys: {
+                    space: {
+                      sys: {
+                        type: 'Link',
+                        linkType: 'Space',
+                        id: '2bg01iqy0isv',
+                      },
+                    },
+                    id: '63aHMAZD0NxIREYx675tYq',
+                    type: 'Entry',
+                    createdAt: '2023-09-13T13:12:27.770Z',
+                    updatedAt: '2023-09-14T14:29:33.822Z',
+                    environment: {
+                      sys: {
+                        id: 'testing',
+                        type: 'Link',
+                        linkType: 'Environment',
+                      },
+                    },
+                    revision: 2,
+                    contentType: {
+                      sys: {
+                        type: 'Link',
+                        linkType: 'ContentType',
+                        id: 'displayParameters',
+                      },
+                    },
+                    locale: 'en-US',
+                  },
+                  fields: {
+                    title: 'Littérature',
+                    layout: 'two-items',
+                    minOffers: 1,
+                  },
+                },
+              ],
+            },
+          })
+        )
+      })
+    )
+
+    it('should return correct data', async () => {
+      const result = await fetchGTLPlaylists({
+        position: {
+          latitude: 2,
+          longitude: 2,
+        },
+        isUserUnderage: false,
+      })
+
+      expect(result).toEqual([
+        { layout: 'two-items', offers: { hits: [] }, title: 'Jeunesse' },
+        { layout: 'two-items', offers: { hits: [] }, title: 'Littérature' },
+      ])
+    })
+  })
+
+  describe('fetchOffersFromGTLPlaylist', () => {
+    const mockMultipleQueries = jest.spyOn(multipleQueries, 'multipleQueries').mockResolvedValue([
+      {
+        hits: [],
+      } as unknown as SearchResponse<Offer>,
+    ])
+
+    it('should work', async () => {
```

Comment:
Je pense que ça serait cool de séparer ton test en 2 :
- `should execute Algolia multiple queries to fetch offers`
- `should return a GTL playlist`
ou si tu veux un garder un seul test, plus de précision. Le nom de la fonction est précis mais ça donne plus de contexte si tu lis les tests sans forcément connaître le code.